AWSAppSyncClient.xcodeproj/project.pbxproj

```diff
@@ -2387,7 +2387,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				GCC_C_LANGUAGE_STANDARD = gnu11;
 				INFOPLIST_FILE = AWSAppSyncTestAppUITests/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 12.1;
```

Comment:
You've downgraded some targets from 12.**1** to 12.**0**. 

Just to be sure, did you validate that these changes are safe?

Reply:
Here the `AWSAppSyncTestAppUITests` is an empty test project. I've verified all the other downgraded targets are success to build.